### ISSUE TITLE
R >= 4.1 ?

### ISSUE THREAD

**Issue by numpde** [neurogenomics/ewceData#1]
I'm trying to 

`devtools::install_github("neurogenomics/ewceData")`

and get

```
Downloading GitHub repo neurogenomics/ewceData@HEAD
✓  checking for file ‘/tmp/RtmpSbzz2d/remotes1d59354cba42a1/neurogenomics-ewceData-fdc3feb/DESCRIPTION’ ...
─  preparing ‘ewceData’:
✓  checking DESCRIPTION meta-information ...
─  checking for LF line-endings in source and make files and shell scripts
─  checking for empty or unneeded directories
─  building ‘ewceData_0.99.7.tar.gz’
   
Installing package into ‘.../R/x86_64-pc-linux-gnu-library/4.0’
(as ‘lib’ is unspecified)
ERROR: this R is version 4.0.4, package 'ewceData' requires R >= 4.1
```

--

I was able to install after modifying the [DESCRIPTION](https://github.com/neurogenomics/ewceData/blob/main/DESCRIPTION#L27) file.
Same issue and fix with [EWCE](https://github.com/NathanSkene/EWCE).

**Comment by NathanSkene** [comment]
Hi, we've just had to add this dependency so that we can get the package back on bioconductor, apologies for the inconvenience.

@Al-Murphy can you explain best solution for this?

**Comment by Al-Murphy** [comment]
Hi,

R version 4.1 is necessary as the data is only available on the ExperimentHub database snapshot for R version 4.1. The snapshot of the database available for R 4.0 won't contain EWCE's data. This data is used both for the vignette and for internal EWCE functions so the package's functionality can't be used without it. Unfortunately, you will therefore have to install R version 4.1. The package should go live on Bioconductor around late April which will make things easier but in the meantime, if you have issues installing any dependencies on R 4.1 feel free to reach out.

**Comment by numpde** [comment]
Thank you for the explanation. Indeed, with the `R >= 4.0` hack, I was getting

```
Error in ewceData::mouse_to_human_homologs() :
  resource mouse_to_human_homologs not found in ExperimentHub 
```

**Comment by Al-Murphy** [comment]
Hi @numpde, the data has been removed from the git history of EWCE as per a requirement of Bioconductor. EWCE will be reinstated on Bioconductor in the next release (likely April/May this year). To make EWCE usable for those with R < 4.1 in the meantime, we have made a separate repository based on a previous commit of EWCE and added back in the necessary data. See [neurogenomics/EWCE](https://github.com/neurogenomics/EWCE) for details and feel free to create issues on that repo if anything is wrong.
